app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  # Only the autenticated users are able to navigate  in the app
+  before_action :authenticate_user!
+
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
@sansuaza could you please explain the **rescue_from**

Reply:
that is the way to manage the access denied error, in this case the user is redirected to another page